### PR TITLE
Updated Information Architecture. Revised README / CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by brooklynrob**
This pull request:

- Updates and re-organizes the README.md, and also moves content about how to contribute to .github/CONTRIBUTING.md
- Updates top and side navigate to 1) Add GitLab along with Github to top, 2) Add About to Top; 3) Create an about section of side menu
- Starting point folders of "About" md files (timeline, collaborate, faq, etc.)
- Starting point folder of "Meeting Minutes" including starting point notes for the 2019.1.29 kick-off minutes (for @aitana16 to complete)

Pasted_Image_1_30_20__9_59_PM

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,4 +1,6 @@
 .DS_Store
+yarn.lock
```
Comment (maoo) [inline .gitignore]: @brooklynrob - please remove this entry, we want `yarn.lock` to be pushed into GitHub, see https://legacy.yarnpkg.com/blog/2016/11/24/lockfiles-for-all/

File: .travis.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ node_js:
   - '8'
 branches:
   only:
-    - master
+    - features/rfu001
```
Comment (maoo) [inline .travis.yml]: @brooklynrob  - please revert this to `master`

File: website/siteConfig.js [hunk 3]
```diff
@@ -17,16 +17,18 @@ const alloyHighlighter = require('./alloyHighlighter');
 const siteConfig = {
   title: 'alloy', // Title for your website. THIS CHANGES THE TITLE TAGS FOR THE WEBSITE (LOOK IN BROWSER TAB AT TOP)
   tagline: 'Language and visual platform to generate models for the financial services industry', //THIS CHANGES THE TITLE TAGS FOR THE WEBSITE (LOOK IN BROWSER TAB AT TOP)
-  cname: 'alloy.finos.org',
-  url: 'https://alloy.finos.org/',
-  baseUrl: '/',
+  // cname: 'alloy.finos.org',
```
Comment (maoo) [inline website/siteConfig.js]: @brooklynrob - please revert this

File: website/siteConfig.js [hunk 4]
```diff
@@ -17,16 +17,18 @@ const alloyHighlighter = require('./alloyHighlighter');
...
+  url: 'https://brooklynrob.github.io',
+  baseUrl: '/purealloy',
```
Comment (maoo) [inline website/siteConfig.js]: @brooklynrob - please revert this

File: website/siteConfig.js [hunk 5]
```diff
@@ -17,16 +17,18 @@ const alloyHighlighter = require('./alloyHighlighter');
 const siteConfig = {
   title: 'alloy', // Title for your website. THIS CHANGES THE TITLE TAGS FOR THE WEBSITE (LOOK IN BROWSER TAB AT TOP)
   tagline: 'Language and visual platform to generate models for the financial services industry', //THIS CHANGES THE TITLE TAGS FOR THE WEBSITE (LOOK IN BROWSER TAB AT TOP)
-  cname: 'alloy.finos.org',
-  url: 'https://alloy.finos.org/',
-  baseUrl: '/',
+  // cname: 'alloy.finos.org',
+  url: 'https://brooklynrob.github.io',
```
Comment (maoo) [inline website/siteConfig.js]: @brooklynrob - please revert this

**Comment by maoo** [comment]
@brooklynrob - I've reviewed the PR and dropped you comments.

@gs-bracej - can you please review content changes?

File: .gitignore (hunk 1 above)
Comment (brooklynrob) [reply .gitignore]: Changed back and added as a new commit.

**Comment by maoo** [comment]
LGTM, ty @brooklynrob . Merging.

**Pull request merged**